contracts/src/plugin/extensions/proposal/Proposal.sol

```diff
@@ -2,82 +2,32 @@
 
 pragma solidity ^0.8.8;
 
-import {Counters} from "@openzeppelin/contracts/utils/Counters.sol";
 import {ERC165} from "@openzeppelin/contracts/utils/introspection/ERC165.sol";
 
-import {IDAO} from "../../../dao/IDAO.sol";
 import {IProposal} from "./IProposal.sol";
 
 /// @title Proposal
 /// @author Aragon X - 2022-2023
 /// @notice An abstract contract containing the traits and internal functionality to create and execute proposals that can be inherited by non-upgradeable DAO plugins.
 /// @custom:security-contact sirt@aragon.org
 abstract contract Proposal is IProposal, ERC165 {
-    using Counters for Counters.Counter;
-
-    /// @notice The incremental ID for proposals and executions.
-    Counters.Counter private proposalCounter;
-
     /// @inheritdoc IProposal
-    function proposalCount() public view override returns (uint256) {
-        return proposalCounter.current();
+    function proposalCount() public pure override returns (uint256) {
+        return type(uint256).max;
```

Comment:
We probably should add here a comment to explain why we are doing this now.

Reply:
Description is added in `IProposal` which is clear as this function has `inheritdoc` above which dev should go and check in IProposal anyways.

Reply:
Even if we added a `@dev` comment, this function would be returning a 100% incorrect value. Any contract/UI depending on it would incorrectly think that things are still working while `2^256-1` is not.

In terms of reliability, I would either:
- Remove the function or 
- Make it `revert()` to signal that this function is no longer valid

This is a breaking change regardless